test/jb_token_store/change_for.test.js

```diff
@@ -0,0 +1,94 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+
+import jbDirectory from '../../artifacts/contracts/JBDirectory.sol/JBDirectory.json';
+import jbOperatoreStore from '../../artifacts/contracts/JBOperatorStore.sol/JBOperatorStore.json';
+import jbProjects from '../../artifacts/contracts/JBProjects.sol/JBProjects.json';
+import jbToken from '../../artifacts/contracts/JBToken.sol/JBToken.json';
+import { Contract } from 'ethers';
+
+describe('JBTokenStore::changeFor(...)', function () {
+  const PROJECT_ID = 2;
+
+  async function setup() {
+    let [deployer, ...addrs] = await ethers.getSigners();
```

Comment:
Add "owner" and "newOwner" to clean code of unnecessary variable uses.

Reply:
`owner` isn't used in this test suite and `newOwner` is used in one test so I think it should be fine to define it once in the local scope

Reply:
This definitely applies to other files though